### PR TITLE
feat(tutorial): add report generation from pipeline

### PR DESCRIPTION
**Pull request by olivierabz**

### CODE REVIEW

File: notebooks/Tutorial4_Client_side_processors.ipynb [hunk 1]
```diff
@@ -965,8 +1015,19 @@
...
+  },
   "language_info": {
-   "name": "python"
+   "name": "python",
+   "version": "3.9.10 (main, Feb 15 2022, 10:33:32) \n[Clang 13.0.1 ]"
+  },
+  "vscode": {
+   "interpreter": {
+    "hash": "f00a74d3855d69ea87fc244e284c76fff9b0b06e2abfa5ec036d651dcf738e6e"
+   }
```
Comment (tcrasset) [inline notebooks/Tutorial4_Client_side_processors.ipynb:1030]: Hmm, the cleaners did not run :/

**Review by tcrasset — APPROVED** [review]
Approving for the code, but let's first fix the cleaning issue. We should not see metadata output in the notebooks.

**Comment by tcrasset** [comment]
Also, you added a feature, the commit message should be `feat(tutorial)` instead of `fix(tutorial)`.

**Review by jpetot — APPROVED** [review]
looks perfect, have you run tutorial tests?

**Pull request merged**